harbor-ui/src/main.rs

```diff
@@ -448,8 +448,14 @@ impl HarborWallet {
                         let amount = if self.is_max {
                             None
                         } else {
-                            // TODO: error handling
-                            Some(self.send_amount_input_str.parse::<u64>().unwrap())
+                            match self.send_amount_input_str.parse::<u64>() {
+                                Ok(amount) => Some(amount),
+                                Err(e) => {
+                                    eprintln!("Error parsing amount: {e}");
```

Comment:
nice I can show a toast for these parse fails. I also need to fix the amount input so you can only type numbers

Reply:
> I also need to fix the amount input so you can only type numbers

i tried to do this but couldnt figure it out